.github/workflows/build.yaml

```diff
@@ -49,10 +49,9 @@ jobs:
   build-and-lint:
     needs:
       generate-client-files
-    uses: GEWIS/actions/.github/workflows/typescript-lint-and-build.yml@v0.0.2
+    uses: GEWIS/actions/.github/workflows/lint-and-build-yarn.yml@v1

```

Comment:
Why are @JustSamuel's commits in your PR? Something seems to be going wrong, so can you rebase such that this PR only includes changes to the orders?

Reply:
No idea, I think I probably messed something up while rebasing, @Gijsdeman was so lovely to fix it for me!